pkg/satellite/apikeys.go

```diff
@@ -24,15 +30,47 @@ type APIKeys interface {
 	Delete(ctx context.Context, id uuid.UUID) error
 }
 
-// APIKey describing apikey model in the database
+// APIKey describing api key model in the database
 type APIKey struct {
 	ID uuid.UUID `json:"id"`
 
 	// Fk on project
 	ProjectID uuid.UUID `json:"projectId"`
 
-	Key  []byte `json:"key"`
-	Name string `json:"name"`
+	Key  MockKey `json:"key"`
```

Comment:
I wonder whether the Key should be part of this struct. In many places it's being overwritten by zeros. Maybe rename this as `APIKeyInfo` and then all the places that need that Key it is returned outside of the struct.

This should avoid accidentally leaking the Key and might remove some code.